docs/articles/WaCore.Data.Ef.md

```diff
@@ -0,0 +1,47 @@
+﻿---
+uid: UnitOfWork_Module
+---
+
+## 1. What is it?
+* Implementation of UnitOfWork pattern
+* While using the Repository pattern
+* Using Entity Framework Core 2.0
+
+
+## 2. Features
+* UoW is responsible for creating repositories
+* Not depending on any specific DbContext, you can create your own
+
+
+## 3. How to find it
```

Comment:
Let's replace this section with 'How to install' or 'NuGet package'

Reply:
Changed